libraries/app/util.cpp

```diff
@@ -34,9 +34,22 @@ using boost::multiprecision::uint256_t;
 
 static fc::uint128_t to_capped_128( const uint256_t& t )
 {
+#if defined(__OpenBSD__)
+   const fc::uint128_t max = std::numeric_limits<fc::uint128_t>::max();
```

Comment:
I think this is not an OpenBSD problem but a Clang problem. @nathanhourt ran into it as well. He said it can be fixed by enabling compiler extensions in CMakeLists.txt.

Reply:
See #1941

Reply:
Exactly. The change is now gone.